Add avr32 definitions to fdisk, these definitions exist in current
util-linux sources.

Index: busybox-1.14.2/util-linux/fdisk_osf.c
===================================================================
--- busybox-1.14.2.orig/util-linux/fdisk_osf.c	2009-08-08 21:12:19.000000000 -0700
+++ busybox-1.14.2/util-linux/fdisk_osf.c	2009-08-08 21:13:36.000000000 -0700
@@ -45,7 +45,7 @@
 
 #if defined(i386) || defined(__sparc__) || defined(__arm__) \
  || defined(__m68k__) || defined(__mips__) || defined(__s390__) \
- || defined(__sh__) || defined(__x86_64__)
+ || defined(__sh__) || defined(__x86_64__) || defined(__avr32__)
 #define BSD_LABELSECTOR   1
 #define BSD_LABELOFFSET   0
 #elif defined(__alpha__) || defined(__powerpc__) || defined(__ia64__) \
